recipe/meta.yaml

```diff
@@ -1,52 +1,105 @@
 {% set name = "mesaLib" %}
-{% set version = "18.0.0" %}
+{% set version = "21.2.5" %}
+{% set build = 0 %}
```

Comment:
Can we remove the build number indirection?

Reply:
Done in 6168c0e